soh/soh/Enhancements/randomizer/option_descriptions.cpp

```diff
@@ -2,30 +2,30 @@
 
 namespace Rando {
 void Settings::CreateOptionDescriptions() {
-    mOptionDescriptions[RSK_FOREST] = "Closed - Kokiri sword & shield are required to access "
+    mOptionDescriptions[RSK_FOREST] = "Closed - Kokiri Sword & Shield are required to access "
```

Comment:
Should mention it's the Deku Shield specifically, like the tooltip for Closed Deku.